inc/abstractitiltarget.class.php

```diff
@@ -96,9 +96,9 @@ abstract protected function getItem_Supplier();
     * Get an instance object for the relation between the target itemtype
     * and an object of any itemtype
     *
-    * @return CommonDBTM
+    * @return CommonDBRelation
     */
-   abstract public function getItem_Item();
+   abstract public static function getItem_Item(): CommonDBRelation;
```

Comment:
switching to static method to avoif creating an object for this call which does not uses $this